README.md

```diff
@@ -7,22 +7,22 @@ useful for testing downstream workflows.
 ## Downloads
 
 - [Tiny](https://github.com/smart-on-fhir/sample-bulk-fhir-datasets/archive/refs/heads/10-patients.zip)
-  (10 patients, 1.7MB zipped, 17MB unzipped)
+  (10 patients, 1.9MB zipped, 19MB unzipped)
 - [Small](https://github.com/smart-on-fhir/sample-bulk-fhir-datasets/archive/refs/heads/100-patients.zip)
-  (100 patients, 15MB zipped, 117MB unzipped)
+  (100 patients, 17MB zipped, 129MB unzipped)
 - [Medium](https://github.com/smart-on-fhir/sample-bulk-fhir-datasets/archive/refs/heads/1000-patients.zip)
-  (1,000 patients, 162MB zipped, 1.2GB unzipped)
-- [Large](https://github.com/smart-on-fhir/sample-bulk-fhir-datasets/archive/refs/heads/10000-patients.zip)
-  (10,000 patients, 1.7GB zipped, 13GB unzipped)
```

Comment:
yeah this one is overkill.

dumb semantics - does that make the 1k one the large one?

Reply:
Oh yeah thanks - can just be S/M/L now